### DISCUSSION TITLE
Error: failed to update dependencies: Internal server error

### DISCUSSION THREAD

**Discussion by Hrushi11** [deta/cloud-docs#316] [Help] [answered]
My `requirements.txt` file includes `tensorflow` package. Once I take this requirement out the build works properly. Does deta not support tensorflow..

```
Deploying...
Successfully deployed changes
Updating dependencies...
Error: failed to update dependencies: Internal server error
```

Please let me know how I can resolve this or deta has some restrictions in requirements that we can include.

**Comment by abdelhai** [comment, marked as answer]
hi, tf size exceeds our limits and can't run on deta micros yet https://docs.deta.sh/docs/micros/about#important-notes (7-8)

**Comment by Hrushi11** [comment]
> hi, tf size exceeds our limits and can't run on deta micros yet https://docs.deta.sh/docs/micros/about#important-notes (7-8)

Thanks a lot, Got it!

**Reply by qaixerabbas** [reply]
hi everyone, is there any alternative to deta that supports TensorFlow ? I am trying to deploy a basic image classification API and need a simple solution like deta.

**Reply by Hrushi11** [reply]
For the free tier there is none. If you want for limited time then you could use the Amazon's EC2 instance by signing up.
